src/types/api/nodes.ts

```diff
@@ -62,6 +65,20 @@ export interface TSystemStateInfo {
     /** double */
     MaxDiskUsage?: number;
     Location?: TNodeLocation;
+
+    /**
+     * int64
+     *
+     * a positive value means the peer is ahead in time; a negative value means it's behind
+     */
+    MaxClockSkewWithPeerUs?: string;
+    MaxClockSkewPeerId?: number;
+
+    /** uint64 */
+    DisconnectTime?: string;
```

Comment:
Values that are new in backend proto file. They received from backend, but are not used in current PR. Added to `interface` to keep it relevant